### PR TITLE
httpie 1.0.3

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by javian** [comment]
Awaiting further updates before merging.

**Comment by jeffbyrnes** [comment]
FYI, upstream changes were accepted by the httpie maintainer.

**Comment by chenrui333** [comment]
Cool, thanks!

**Review by jeffbyrnes — CHANGES_REQUESTED** [review]

File: Formula/httpie.rb
Comment (SMillerDev) [reply Formula/httpie.rb]: Probably even easier to just use the url of the upstream patch. For example: https://github.com/Homebrew/homebrew-core/blob/master/Formula/upx.rb#L21-L24

Comment (chenrui333) [reply Formula/httpie.rb]: Updated with using the online commit hash.

**Comment by SMillerDev** [comment]
Why does it need to patch a file that generates the formula in the formula? It won't affect any usage of httpie right?

**Comment by javian** [comment]
There's a script that can extract the dependencies https://pypi.python.org/pypi/homebrew-pypi-poet which are then inserted into the formula and this is what needs to be done here and all other formulae constructed this way.

**Comment by jeffbyrnes** [comment]
So yeah, the patch won’t work that way, since this is a generated binary we’re installing.

So either @javian’s suggestion, or manually copying in the new reps, is what’s needed here.

@chenrui333 if you want to close this, I’m happy to take care of it instead!

**Comment by fxcoudert** [comment]
@jeffbyrnes we'll take you up on that offer :)

**Pull request closed without merging**
